fuzz/parse.cc

```diff
@@ -25,6 +25,19 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
     out_url->set_pathname(input);
     out_url->set_search(input);
     out_url->set_hash(input);
+    out_url->set_port(input);
+
+    // getters
+    out_url->get_protocol();
```

Comment:
I am concerned that these function calls might be optimized away in some way since they may have no side-effect?

We might avoid that with something like this...
```C++
volatile size_t length = 0; // volatile forces the compiler to store the results without undue optimizations
length += out_url->get_protocol().size();
...
```

Reply:
Ah, you're right. Thanks!